localstack/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.4.1.dev"
+__version__ = "1.4.1.dev1"
```

Comment:
i think this should stay `1.4.1.dev` since it indicates a placeholder, whereas `dev1` is a concrete release number.

Reply:
Thanks! That was actually a leftover from the initial publishing of `localstack-core` to resolve the cyclic dependency in the pipeline.